app.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
+var pictureBlockEl = document.getElementById('productPictures');
+var left = document.getElementById('left');
+var center = document.getElementById('center');
+var right = document.getElementById('right');
+
 var totalClicks = 0;
-var randomNumArray = [];
+var clickLimit = 25;
+var clickTotals = [];
+var viewTotals = [];
 
-function RandomPicture(name, src){
+var currentlyShowing = [];
+
+function RandomPicture(name, src, clicks, views){
```

Comment:
I would maybe call this just Picture, because each individual picture isn't random, it's just that the one you choose to show is chosen at random.